org.eclipse.xtext.ui/src/org/eclipse/xtext/ui/resource/SourceAttachmentPackageFragmentRootWalker.java

```diff
@@ -296,7 +296,7 @@ protected T handle(IJarEntryResource jarEntry, TraversalState state) {
 	 */
 	private String getBundleSymbolicName(Manifest manifest) {
 		String name = manifest.getMainAttributes().getValue("Bundle-SymbolicName");
-		if (name != null) {
+		if (name != null && !name.trim().isEmpty()) {
```

Comment:
I think that instead of trimming and check if empty you could use https://docs.oracle.com/en/java/javase/11/docs/api/java.base/java/lang/String.html#isBlank()

Reply:
I wanted to keep it compatible to Java 8, but if Java 11 is the baseline, I can of course change it.